tests/lint/test_run.py

```diff
@@ -0,0 +1,56 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/pylint-dev/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/pylint-dev/pylint/blob/main/CONTRIBUTORS.txt
+
+from io import BufferedReader
+from pathlib import Path
+from typing import Any
+from unittest.mock import MagicMock, mock_open, patch
+
+import pytest
+
+from pylint import lint
+from pylint.testutils.utils import _test_cwd
+
+
+@pytest.mark.parametrize(
```

Comment:
Why isn't this grouped with the other tests? I don't think we need a separate file as we don't follow the "file + test_file" structure

Reply:
I had a bit of confusion around using protected members in tests python. Figured it out and merged the tests back into the other file